pyproject.toml

```diff
@@ -42,10 +42,10 @@ keywords = [
 ]
 
 dependencies = [
-    "transformers == 4.39.3",
-    "torch-xla>=2.2.0",
-    "torch>=2.2.0",
-    "loguru == 0.6.0"
+    "transformers",
```

Comment:
removing versions mean that we will hit incompatibilities as APIs change (I have already hit that with transformers). I am not sure it is a good idea.